admin/commands/base.go

```diff
@@ -167,7 +171,7 @@ var customLabelRE = regexp.MustCompile(`^([a-zA-Z_][a-zA-Z0-9_]*)=([^='", ]+)$`)
 // ParseCustomLabels parses --custom-labels flag value.
 //
 // Note that quotes around value are parsed and removed by shell before this function is called.
-// E.g. the value of [[--custom-labels='region=us-east1, mylabel=mylab-22']] will be received by this function
+// E.g. The value of [[--custom-labels='region=us-east1, mylabel=mylab-22']] will be received by this function
 // as [[region=us-east1, mylabel=mylab-22]].
 func ParseCustomLabels(labels string) (map[string]string, error) {
```

Comment:
instead of this function we can use maps https://github.com/alecthomas/kong#maps